cluster/plan.go

```diff
@@ -764,7 +764,7 @@ func getTagMajorVersion(tag string) string {
 }
 
 func getCloudConfigChecksum(config v3.CloudProvider) string {
-	configByteSum := md5.Sum([]byte(fmt.Sprintf("%s", config)))
+	configByteSum := md5.Sum([]byte(fmt.Sprintf("%v", config)))
```

Comment:
can you change the checksum for the cloud config to be like v0.2 https://github.com/rancher/rke/blob/master/cluster/plan.go#L832-L835

Reply:
do you mean to cherry-pick 66fb2c4a as well? but should that be in a separate PR?